docs/source/FPGA-to-bitstream/Yosys compilation.rst

```diff
@@ -13,21 +13,21 @@ To build you may use the Makefile wrapper in the clone repository (https://githu
 User guide
 ----------
 
-A pre-defined Yosys TCL script is under ``$FAB_ROOT/nextpnr/fabulous/synth/synth_fabulous_dffesr.tcl`` for FABulous version3. If the output file, denoted below as ``<JSON_or_BLIF_file>``, has a ``.blif`` file extension, then output will be produced in the Berkeley Logic Interchange Format (BLIF) and will be synthesised appropriately for the VPR flow. Otherwise, output will be produced as JSON, synthesised for the nextpnr flow.
+FABulous is supported by upstream Yosys, using the ``synth_fabulous`` pass. First, run ``yosys``, which will open up an interactive Yosys shell. If synthesizing for the nextpnr flow, run the following commands:
 
 .. code-block:: console
 
-	yosys -p "tcl <path/to/TCL/File>/synth_fabulous_dffesr.tcl <K-LUT> <top_module> <JSON_or_BLIF_file>" <benchmark_netlist>
```

Comment:
personally, I'd recommend using it non-interactively like
`yosys -p "synth_fabulous -top <toplevel> -json <out.json>" <files.v>`

as this is less typing and easier to include in scripts (like our examples).

Reply:
Agreed! I'll get that changed.